Neos.Flow.Log/Classes/FormatUtility.php

```diff
@@ -0,0 +1,67 @@
+<?php
+namespace Neos\Flow\Log;
+
+/**
+ *
```

Comment:
Some comment, maybe?

Also, the name `FormatUtility` sounds rather generic. Something completely different would be a `FormattedVariable` class, where you pass the `$variable` to the constructor and then call `getAsPlaintext()` on that. Phew. :)

Speaking of which, I think it should be "plain text", not "plaintext".